lib/crypto.js

```diff
@@ -647,9 +647,21 @@ exports.setEngine = function setEngine(id, flags) {
   return binding.setEngine(id, flags);
 };
 
-exports.randomBytes = exports.pseudoRandomBytes = randomBytes;
-
-exports.rng = exports.prng = randomBytes;
+exports.randomBytes = util._deprecateSync(randomBytes,
```

Comment:
Looks fine to me if it works. I'd reword to 'Using randomBytes with a callback is deprecated.' Also, let's use a named function to help on stack traces (`function randomBytes()`)